api/swagger/swagger.yaml

```diff
@@ -810,6 +827,11 @@ definitions:
         type: string
       name:
         type: string
+      url:
+        type: string
+        description: URL to source's website
+      terms_and_conditions_url:
```

Comment:
It felt redundant to add a description `URL to the Terms and Conditions` or similar...